src/add_crosswalk.py

```diff
@@ -1,264 +1,383 @@
+            elif len(output_flows.loc[output_flows.From_Node == to_node]['HydroID']) == 1:
+                update_id = output_flows.loc[output_flows.From_Node == to_node]['HydroID'].item()
 
             else:
-                update_id = output_flows.loc[output_flows.HydroID==short_id]['HydroID'].item()
-
-            str_order = output_flows.loc[output_flows.HydroID==short_id]['order_'].item()
-            sml_segs = pd.concat([sml_segs, pd.DataFrame({'short_id':[short_id], 'update_id':[update_id], 'str_order':[str_order]})], ignore_index=True)
-
-    print("Number of short reaches [{} < {} and {} < {}] = {}".format("areasqkm", min_catchment_area, "LengthKm", min_stream_length, len(sml_segs)))
+                update_id = output_flows.loc[output_flows.HydroID == short_id]['HydroID'].item()
+
+            str_order = output_flows.loc[output_flows.HydroID == short_id]['order_'].item()
+            sml_segs = pd.concat(
+                [
+                    sml_segs,
+                    pd.DataFrame(
+                        {'short_id': [short_id], 'update_id': [update_id], 'str_order': [str_order]}
+                    ),
+                ],
+                ignore_index=True,
+            )
+
+    print(
+        f"Number of short reaches [areasqkm < {min_catchment_area} and LengthKm < {min_stream_length}] = "
+        f"{len(sml_segs)}"
+    )
 
     # calculate src_full
-    input_src_base = pd.read_csv(input_srcbase_fileName, dtype= object)
-    if input_src_base.CatchId.dtype != 'int': input_src_base.CatchId = input_src_base.CatchId.astype(int)
+    input_src_base = pd.read_csv(input_srcbase_fileName, dtype=object)
+    if input_src_base.CatchId.dtype != 'int':
+        input_src_base.CatchId = input_src_base.CatchId.astype(int)
 
-    input_src_base = input_src_base.merge(output_flows[['ManningN','HydroID','NextDownID','order_']],left_on='CatchId',right_on='HydroID')
+    input_src_base = input_src_base.merge(
+        output_flows[['ManningN', 'HydroID', 'NextDownID', 'order_']], left_on='CatchId', right_on='HydroID'
+    )
 
     input_src_base = input_src_base.rename(columns=lambda x: x.strip(" "))
-    input_src_base = input_src_base.apply(pd.to_numeric,**{'errors' : 'coerce'})
-    input_src_base['TopWidth (m)'] = input_src_base['SurfaceArea (m2)']/input_src_base['LENGTHKM']/1000
-    input_src_base['WettedPerimeter (m)'] = input_src_base['BedArea (m2)']/input_src_base['LENGTHKM']/1000
-    input_src_base['WetArea (m2)'] = input_src_base['Volume (m3)']/input_src_base['LENGTHKM']/1000
-    input_src_base['HydraulicRadius (m)'] = input_src_base['WetArea (m2)']/input_src_base['WettedPerimeter (m)']
+    input_src_base = input_src_base.apply(pd.to_numeric, **{'errors': 'coerce'})
+    input_src_base['TopWidth (m)'] = input_src_base['SurfaceArea (m2)'] / input_src_base['LENGTHKM'] / 1000
+    input_src_base['WettedPerimeter (m)'] = (
+        input_src_base['BedArea (m2)'] / input_src_base['LENGTHKM'] / 10009
```

Comment:
@CarsonPruitt-NOAA This extra 9 was causing the incorrect checksums I was seeing earlier today. Seems to be resolved as of the latest commit.

Reply:
Oh wow that was a really good catch. That's a huge typo that I'm sure was affecting our synthetic rating curves. It's in our Manning's equation calculation!